index.html

```diff
@@ -1,55 +1,237 @@
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h1 class="title">Multi-Post Stories Gain+Glory</h1>
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h1 class="title">Multi-Post Stories Gain+Glory</h1>
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h1 class="title">Multi-Post Stories Gain+Glory</h1>
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h1 class="title">Multi-Post Stories Gain+Glory</h1>
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="third-section">
+      <div class="container">
+        <h1>About Me</h1>
+        <p class="about-desc">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="box-button about-btn">Get My Resume</button>
+      </div>
+      <div class="boxes">
+        <div class="about-box">
+          <div><img class="about-img" src="./Images/icon1.svg" alt="" /></div>
+          <div class="about-menu">
+            <h1 class="about-title">Languages</h1>
+            <ul class="about-list">
```

Comment:
- [ ] Dear @danutibe07, according to the main requirements the technology categories ("Languages", "Frameworks" and "Skills") are expected to be in a `ul`, each of them in a `li`. Please modify those dives into a ul and wrap all your  ("Languages", "Frameworks" and "Skills") in a `li`